teachers_behavior.rb

```diff
@@ -0,0 +1,17 @@
+module Salary
```

Comment:
It make sense to organize teacher and apprentice-teacher to a super class but I am confused about when to use a module.☹️